.github/workflows/ci.yml

```diff
@@ -35,21 +60,24 @@ jobs:
         run: |
           curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
           chmod +x ./cc-test-reporter
-          ./cc-test-reporter before-build        
+      - name: Install dependencies
+        run: bundle install
       - name: Setup Database
-        run: bundle exec rails db:create db:migrate
+        run: |
+          bundle exec rake parallel:load_schema
       - name: Check for untracked changes in schema.rb
         uses: rootstrap/check_untracked_changes@v1
         with:
           path: "./db/schema.rb"
-      - name: I18n Health
-        run: bundle exec i18n-tasks health
-      - name: Run Code Analysis
-        run: |
-          bundle exec rake code:analysis
       - name: Run Tests
+        env:
+          KNAPSACK_CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
+          KNAPSACK_CI_NODE_INDEX: ${{ matrix.ci_node_index }}
+          KNAPSACK_LOG_LEVEL: info
+          PARALLEL_TESTS_CONCURRENCY: 2
```

Comment:
is it possible to get this number dynamically from the servers where it's running?